packages/outstatic/package.json

```diff
@@ -50,6 +52,7 @@
     "react-datepicker": "^4.8.0",
     "react-hook-form": "^7.31.3",
     "react-textarea-autosize": "^8.3.4",
+    "schema-to-yup": "^1.12.9",
```

Comment:
I like the use of JSON schema, specifically because it's a format that translates well to & from YAML